Feat (client/src/components/home) create and edit home component

Matias-Rodriguez-UX opened the pull request.

Matias-Rodriguez-UX committed:
Clean Cache

Matias-Rodriguez-UX committed:
add provider on index and modified app.js  and home

Matias-Rodriguez-UX committed:
created reducer, store, import bootstrap

Javier-Pedernera commented:
Mati ten en cuenta que estás mergeando a la rama main. Deberíamos mergear a la rama develop o crear una rama developer para hacer las primeras pruebas

Matias-Rodriguez-UX committed:
Reorder folders components created

Se crearon:
Banner, Navbar, Main, Section1, Why, testimonios, questions, team, contact, footer, shop, join y se cambio el landing
